.github/publish-test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
GitHub Actions may run this with `-e` or equivalent implicitly, but I think we should include it in this script explicitly to make it clearer to the reader that the purpose of the script is to confirm that all of the makefile targets return 0.  `set -eo pipefail`